apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/PoolRewrite/SessionPool.cs

```diff
@@ -64,6 +66,85 @@ public SessionPool(SpannerClient client, SessionPoolOptions options, Logger logg
             _logger = logger ?? Logger.DefaultLogger;
             _detachedSessionPool = new DetachedSessionPool(this);
             _sessionAcquisitionSemaphore = new SemaphoreSlim(Options.MaximumConcurrentSessionCreates);
+            Task.Run(() => PoolMaintenanceLoop(this));
+        }
+
+        /// <summary>
+        /// A long-running loop performing pool maintenance. Each iteration runs <see cref="MaintainPool"/>.
+        /// This is a static method to allow the target pool to be garbage collected, at which point the 
```

Comment:
nit: Incomplete comment "at which point the"...

Reply:
Whoops, thanks - will fix.